.github/labels.yml

```
@@ -0,0 +1,70 @@
+---
+# Rather than breaking up descriptions into multiline strings we disable that
+# specific rule in yamllint for this file.
+# yamllint disable rule:line-length
+- color: "eb6420"
+  description: This issue or pull request is awaiting the outcome of another issue or pull request
+  name: blocked
+- color: "000000"
+  description: This issue or pull request involves changes to existing functionality
+  name: breaking change
+- color: "d73a4a"
+  description: This issue or pull request addresses broken functionality
+  name: bug
+- color: "07648d"
+  description: This issue will be advertised on code.gov's Open Tasks page (https://code.gov/open-tasks)
+  name: code.gov
+- color: "0366d6"
+  description: Pull requests that update a dependency file
+  name: dependencies
+- color: "5319e7"
+  description: This issue or pull request improves or adds to documentation
+  name: documentation
+- color: "cfd3d7"
+  description: This issue or pull request already exists or is covered in another issue or pull request
+  name: duplicate
+- color: "b005bc"
+  description: A high-level objective issue encompassing multiple issues instead of a specific unit of work
+  name: epic
+- color: "000000"
+  description: Pull requests that update GitHub Actions code
+  name: github-actions
+- color: "0e8a16"
+  description: This issue or pull request is well-defined and good for newcomers
+  name: good first issue
+- color: "ff7518"
+  description: Pull request that should count toward Hacktoberfest participation
+  name: hacktoberfest-accepted
+- color: "a2eeef"
+  description: This issue or pull request will add or improve functionality, maintainability, or ease of use
+  name: improvement
+- color: "fef2c0"
+  description: This issue or pull request is not applicable, incorrect, or obsolete
+  name: invalid
+- color: "ce099a"
+  description: This pull request is ready to merge during the next Lineage Kraken release
+  name: kraken 🐙
+- color: "a4fc5d"
+  description: This issue or pull request requires further information
+  name: need info
+- color: "fcdb45"
+  description: This pull request is awaiting an action or decision to move forward
+  name: on hold
+- color: "ef476c"
+  description: This issue is a request for information or needs discussion
+  name: question
+- color: "7b42bc"
```

Preserve existing `terraform` label (see [here](https://github.com/cisagov/terraformer-packer/actions/runs/3463729899/jobs/5784368902#step:3:60) and [here](https://github.com/cisagov/terraformer-packer/actions/runs/3463729899/jobs/5784368902#step:3:90)):

```suggestion
- color: "5c4ee5"
```

---

Can't the `terraform` label take on a new color?  I'm all for keeping existing labels that are not in the upstream `labels.yml`, but I thought that unifying colors across repos was part of the reason for cisagov/skeleton-generic#118.

---

I may have misunderstood @mcdonnnj, but I thought he said (or at least implied) that if the color of the label changes, it will no longer match with any PRs/issues tagged with that label in the past.  If that is not the case, then I agree with you @jsf9k.

---

Ah, that is a subtlety I was not aware of or had forgotten about.  I will happily commit your suggestion if @mcdonnnj tells us that is indeed the case.

---

The only thing that causes a disconnect (that I am aware of) is differing names. Those result in a label deletion+creation. Everything else is an in-place update for a label and should not result in any label filtering breakage. You can see [here](https://github.com/cisagov/cyhy_amis/actions/runs/3463331190/jobs/5783409364#step:3:99) that the `terraform` label is updated in-place and confirm [here](https://github.com/cisagov/cyhy_amis/pulls?q=is%3Apr+is%3Aopen+label%3Aterraform) it still functions as a filter in that repository.

---

_SNAP!!!_

---

Cool, thanks for clarifying.  Not sure how I got the idea that the color changing was an issue.